### DISCUSSION TITLE
How to order query results based on amount of relations?

### DISCUSSION THREAD

**Discussion by notramo** [martenframework/marten#182] [Q&A]
There is an `User` model, and each `User` can have multiple `Post` records. How to query all users ordered by the amount of posts they have?

**Comment by ellmetha** [comment]
Currently, this functionality isn't available, regrettably. It would require query set annotations. For instance, the ability to annotate a query set with a count aggregate and then utilize it (as part of the query set) to order records. This is something we should implement at one point.
